src/sagemaker/image_uri_config/tensorflow.json

```diff
@@ -298,14 +298,17 @@
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
                     "ap-south-1": "520713654638",
+                    "ap-south-2": "772153158452",
```

Comment:
Please remove these new entries from the framework versions whose repository is `sagemaker-tensorflow` and `sagemaker-tensorflow-serving`.

Please also add these regions to the framework versions that have the repo name `tensorflow-inference-eia`.